locations/categories.py

```diff
@@ -185,17 +210,21 @@ def apply_category(category, item):
 
 top_level_tags = [
     "amenity",
+    "clothes",
     "club",
     "craft",
     "emergency",
     "healthcare",
+    "healthcare:speciality",
     "highway",
     "landuse",
     "leisure",
     "man_made",
     "office",
     "public_transport",
     "shop",
+    "social_facility",
+    "social_facility:for",
```

Comment:
Even though these are part of the category, they aren't top level tags. They are secondary.

`clothes=x` is not a valid category in OSM. `shop=clothes` is, and `shop=clothes` + `clothes=x` is.

This list/get_categories should only be returning the minimal category (`shop=clothes`).

Reply:
Thanks, fixed it. Could you please check again to make sure the revised approach is ok.